google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeployAction.java

```diff
@@ -171,6 +171,9 @@ public void processTerminated(final ProcessEvent event) {
       try {
         if (event.getExitCode() == 0) {
           callback.succeeded(new DeploymentRuntimeImpl(deploymentOutput.toString(), version));
+        } else if (cancelled) {
+          // process cancelled (message should never be seen by user)
+          callback.errorOccurred("Deployment process was cancelled.");
```

Comment:
The user will never see this message as far as *this* pull request is concerned. However, in #621, it can actually be seen by the user and is localized there.